packages/core/src/Synchronizer.ts

```diff
@@ -340,13 +366,12 @@ export class Synchronizer extends EventEmitter {
         })
         await this.processEvents(unprocessedEvents)
         await this._db.update('SynchronizerState', {
-            where: {
-                attesterId: this.attesterId.toString(),
-            },
+            where: {},
```

Comment:
This will update all documents in the collection settings `latestCompleteBlock: blockEnd`, not sure if intentional

Reply:
[8645c48](https://github.com/Unirep/Unirep/pull/367/commits/8645c48a6d9482179dba6df2844b02e358ee9f70)